examples/hello_world/src/lib.rs

```diff
@@ -30,20 +30,25 @@ impl oak::Node for Node {
     fn new() -> Self {
         Node
     }
-    fn invoke(&mut self, method_name: &str, request: &mut oak::Reader, response: &mut oak::Writer) {
+    fn invoke(&mut self, method_name: &str, grpc: &mut oak::Channel) {
```

Comment:
grpc -> channel for consistency?

Reply:
Well, this is the grpc channel only though; e.g. for private data store we will need another channel -- how exactly to obtain that channel is TBD, but it will not be the same as the grpc channel.